sips/sip-019/sip-019-token-metadata-update-notifications.md

```diff
@@ -212,6 +212,26 @@ Notifications that do not meet these requirements must be ignored.
 Given these constraints the notifications this SIP proposes should be taken as _hints_ to metadata
 indexers. Metadata indexers are not obliged to follow them.
 
+# Related work
+
+An alternative considered for token metadata update notifications is for them to be transmitted via
+an off-chain notification service such as an official mailing list or a forum post. While this
```

Comment:
You could also have something more automated, like an RSS feed or an IRC bot or something like that.

Reply:
Thanks, good suggestion.

Complimentary to this SIP, btw, I was actually planning on adding something similar to the Hiro API: an ability for users to subscribe to metadata update notifications via websockets/socket.io, so they don't have to do the heavy lifting of listening to node RPCs if they don't already do so.